FirstApiSandbox/Controllers/BookController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using FirstApiSandbox.Model;
+using FirstApiSandbox.Database;
+using FirstApiSandbox.Service;
+using System.Linq;
+
+namespace FirstApiSandbox.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class BookController : ControllerBase
+    {
+        IService bookService = new BookService();
+
+        // GET: api/Book
+        [HttpGet]
+        public IEnumerable<Book> Get()
+        {
+            return bookService.GetBooksfromService();
+        }
+
+        // GET: api/Book/5
+        //[HttpGet("{id}", Name = "Get")]
```

Comment:
Please Remove unused comments You already have it archived in git if you need them